src/pkg/packager/prepare.go

```diff
@@ -143,16 +144,30 @@ func (p *Packager) FindImages(baseDir, repoHelmChartPath string) error {
 
 				var override string
 				var ok bool
+				var kubeVersion *chartutil.KubeVersion
 
 				if override, ok = chartNames[chart.Name]; ok {
 					chart.Name = "dummy"
 				}
 
+				if kubeVersionOverride != "" {
+					verSplit := strings.Split(kubeVersionOverride, ".")
+
+					kubeVersion = &chartutil.KubeVersion{
+						Version: kubeVersionOverride,
+						Major:   verSplit[0],
+						Minor:   verSplit[1],
+					}
```

Comment:
If you are relying on this being in a particular format should we guard it with a regex?